test-jest/cartesian/ZAxis.spec.tsx

```diff
@@ -0,0 +1,17 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+import { Surface, ZAxis } from '../../src';
+
+describe('<ZAxis />', () => {
+  it("Don't render anything", () => {
+    render(
+      <Surface width={500} height={500}>
+        <ZAxis dataKey="x" name="stature" unit="cm" />
+      </Surface>,
+    );
+    const svg = document.querySelector('svg');
+
+    expect(svg).toBeInTheDocument();
+    expect(svg?.children).toHaveLength(2);
```

Comment:
Why is this condition different to the one in the original test? 

```
    expect(wrapper.find('svg').children.length).to.equal(1);
    expect(wrapper.find('svg noscript').children.length).to.equal(1);
```

Reply:
Yeah I tested the output as well - there is no way this test was working properly beforehand. This change works - it just renders an empty `title` and `desc` tag